src/utils/index.ts

```diff
@@ -186,9 +175,9 @@ export function replaceTextExtracts(
 
 export function replaceUrl(input: string): string {
   const urlRegex =
-    /(\b(https?|ftp|file):\/\/[-A-Z0-9+&@#/%?=~_|!:,.;]*[-A-Z0-9+&@#/%=~_|])/gi;
+    /(?:https?:\/\/|www\.)?[-a-zA-Z0-9@:%._+~#=]{1,256}\.(xn--)?[a-z]{2,20}\b([-a-zA-Z0-9@:%_+[\],.~#?&/=]*[-a-zA-Z0-9@:%_+~#?&/=])*/g;
```

Comment:
Got this regular expression from UIKit https://github.com/sendbird/sendbird-uikit-react/blob/main/src/modules/Message/utils/tokens/tokenize.ts#L54